tests/alwaysOnTop.test.js

```diff
@@ -64,16 +98,102 @@ describe('Tests for Always on top', () => {
         });
     });
 
-    it('should change the always on top property', () => {
-        return app.browserWindow.setAlwaysOnTop(true);
+    it('should toggle the always on top property to true', (done) => {
+        if (isMac) {
+            robot.setMouseDelay(200);
+            robot.moveMouse(190, 0);
+            robot.mouseClick();
+            for (let i = 0; i < 8; i++) {
+                robot.keyTap('down');
+            }
+            robot.keyTap('enter');
+            setTimeout(() => {
+                done();
+            }, 5000)
+        } else {
+            app.browserWindow.getBounds().then((bounds) => {
+                robot.setMouseDelay(200);
+                let x = bounds.x + 95;
+                let y = bounds.x + 95;
+                robot.moveMouse(x, y);
```

Comment:
Are we sure this works consistently and predictably across both Mac and Windows and when the window is resized, etc.?